gap/congruences/conglatt.gi

```diff
@@ -530,10 +521,10 @@ function(poset, opts)
       fi;
     od;
   else
-    symbols := List([1 .. Length(poset)], String);
+    symbols := List([1 .. Length(out_nbs)], String);
   fi;
 
-  rel := List([1 .. Length(poset)], x -> Filtered(poset[x], y -> x <> y));
+  rel := List([1 .. Length(out_nbs)], x -> Filtered(out_nbs[x], y -> x <> y));
```

Comment:
Change to `rel := InNeighbours(DigraphReflexiveTransitiveReduction(poset));`

The remove the line `j := Difference(rel[i], Union(rel{rel[i]}));`.